library/navbar/navbar.css

```diff
@@ -83,4 +83,114 @@
     .dui-nav__search {
         display: none;
     }
-}
\ No newline at end of file
+}
+
+/* NavBar with Search and Actions Starts */
+
+.dui-nav-sch-act {
+    width: 100%;
+    /* 2px for border */
+    height: calc(100% - 2px);
+    box-shadow: 0px 2px 0px 0px var(--dui-primary-color-p2-lgt-35pct);
+
+    background-color: var(--dui-light-theme-bg);
+    display: grid;
+    grid-template-columns: 1fr 2fr 2fr;
+    justify-items: center;
+    align-items: center;
+}
+
+.dui-nav-sch-act__search {
+    width: 95%;
+}
+
+.dui-nav-sch-act__logo {
+    margin: auto;
+}
+
+.dui-nav-sch-act__actions {
+    display: flex;
+    align-items: center;
+    justify-content: center
+}
+
+.dui-nav-sch-act__actions>*:not(:last-child) {
+    margin-right: 3rem;
+}
+
+.dui-nav-sch-act__auth-actions>*:not(:last-child) {
+    margin-right: 2rem;
+}
+
+.dui-nav-sch-act__signup-btn {}
+
+.dui-nav-sch-act__login-btn {}
```

Comment:
remove unused classes. If they are for future reference do mention it in with the commented line